Source/Initialization/InjectorMomentum.H

```diff
@@ -86,6 +87,66 @@ private:
     amrex::Real m_ux_th, m_uy_th, m_uz_th;
 };
 
+namespace {
```

Comment:
I had to use an anonymous namespace here. Otherwise, when compiling, the linker says that the function `generateGaussianFluxDist` is defined multiple times (since this `.H` file is included is several of the `.cpp` files). I am not sure if there is a better way to do this.